api

```diff
@@ -29,6 +29,10 @@ status_green() { #announce the success of a major action
   echo -e "\e[92m$1\e[0m" 1>&2
 }
 
+debug() { #an echo command that only runs when debug mode is on
+  [ "$pi_apps_debug" = true ] && echo "$1"
+}
+
```

Comment:
I am fine if we want to remove this function. I thought it was just nice to have rather than commenting out `echo` commands that we have littered throughout our scripts.